### ISSUE TITLE
Add documentation for the extension

### ISSUE THREAD

**Issue by ocots** [control-toolbox/CTBase.jl#188]
See https://github.com/control-toolbox/CTDirect.jl/blob/v0.9.0/docs/src/api.md

**Comment by jbcaillau** [comment]
@ocots what do you mean? do we agree that adding a dummy function (to be extended in the package extension) allows to provide the docstring with it?

**Comment by ocots** [comment]
I mean that for the moment we have this: https://control-toolbox.org/CTBase.jl/dev/api-plot.html

**Comment by PierreMartinon** [comment]
Yes, the problem is that Documenter does not yet recognize extensions properly, in particular **autodocs** doesn't pick up the extended functions. A possible workaround is to manually add the list of functions to document in a **docs** block. See https://github.com/JuliaDocs/Documenter.jl/issues/2124

When extending from another package, there is the additional problem that the function is not caught by **index** either. In *CTDirect* I added the origin module, which does find the function, however now we also display the doc for the original (before extension) function, which ideally we would hide.

**Comment by ocots** [comment]
I tried but didn't manage to make it work for the plot extension.

**Comment by ocots** [comment]
If I do not manage to do it, I think it is because I extend a a function from another package and not because we have an extension inside our package. Because, here also it does not work: https://github.com/control-toolbox/OptimalControl.jl/issues/249.
